web_site/domain/user.py

```diff
@@ -0,0 +1,90 @@
+##############################################################################
+### user.py - User model                                                   ###
+##############################################################################
+#                                                                            #
+#   Description : Defines users of the application                           #
+#                                                                            #
+#   External Calls:                                                          #
+#                                                                            #
+#                                                                            #
+#  History                                                                   #
+#                                                                            #
+#  Name       Date               Update                                      #
+# =======    ==========         ========================================     #
+# Jerry      08/21/2016         Module created                               #
+#                                                                            #
+##############################################################################
```

Comment:
@rolltide0 Yea, we don't own code here. Go ahead and delete this section. I personally feel the "headings" through the file are needless as well so if the group agrees we'd want to delete those as well.